### PR TITLE
NOTASK: Installed Helm plugins and bumped version of node-deploy

### PR DESCRIPTION
**Pull request by roman-chudik-codeit**

### CODE REVIEW

File: node-deploy/Dockerfile [hunk 1]
```diff
@@ -11,12 +11,24 @@ RUN wget -q -O /home/circleci/.local/bin/runIfChanged https://github.com/sgeisba
...
 RUN curl -fsSL https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
 RUN mkdir -p /home/circleci/.local/bin/ && \
     curl -L https://github.com/instrumenta/kubeval/releases/latest/download/kubeval-linux-amd64.tar.gz | tar -C /home/circleci/.local/bin -xzf - kubeval && \
     chmod a+x /home/circleci/.local/bin/kubeval  
 RUN npm install -g zx
+RUN wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 &&\
+    chmod a+x /usr/local/bin/yq
+## Patching outdated kubeval Helm plugin on the fly
+ENV HELM_KUBEVAL_VER=v0.16.1
+RUN mkdir -p /tmp/helm-kubeval /home/circleci/.local/share &&\
```
Comment (sgeisbacher) [inline node-deploy/Dockerfile]: what is `/home/circleci/.local/share` used for? is it the location where helm-plugins are stored?

File: node-deploy/Dockerfile [hunk 2]
```diff
@@ -11,12 +11,24 @@ RUN wget -q -O /home/circleci/.local/bin/runIfChanged https://github.com/sgeisba
...
+    git clone https://github.com/instrumenta/helm-kubeval /tmp/helm-kubeval &&\
+    yq ".version = \"${HELM_KUBEVAL_VER}\"" -i /tmp/helm-kubeval/plugin.yaml &&\
+    sed -i 's/v${version}/${version}/g' /tmp/helm-kubeval/scripts/install.sh
```
Comment (sgeisbacher) [inline node-deploy/Dockerfile:29]: isnt that `v${version}` just only substituting the version in `echo` command? https://github.com/instrumenta/helm-kubeval/blob/master/scripts/install.sh#L5

**Review by sgeisbacher — DISMISSED** [review]
looks good to me!
i have just 2 small questions

File: node-deploy/Dockerfile (hunk 2 above)
Comment (sgeisbacher) [reply node-deploy/Dockerfile:29]: and doesnt it suffice that it reads the version from the plugin.yaml in the line above?

File: node-deploy/Dockerfile (hunk 1 above)
Comment (roman-chudik-codeit) [reply node-deploy/Dockerfile]: In fact they are stored under the /home/circleci/.local/share/helm/plugins path, I have made that path more precise in the next commit

File: node-deploy/Dockerfile (hunk 2 above)
Comment (roman-chudik-codeit) [reply node-deploy/Dockerfile:29]: For the whole plugin installation process the version number substitution is sufficient, but having double prefix like "vvX.X.X" in version number on the output is not so cool imho :)

Comment (sgeisbacher) [reply node-deploy/Dockerfile:29]: haha 😄  ok, that makes sense 👍

**Review by sgeisbacher — APPROVED** [review]
LGTM 👍

**Pull request merged**